README.md

```diff
@@ -11,24 +8,23 @@ A Neural Net providing a subgrid parameterization of atmospheric convection in a
 The work is described in a [GRL paper](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2020GL091363) with [accompanying code available](https://github.com/yaniyuval/Neural_nework_parameterization/tree/v.1.0.3). The repository contains the neural net and its implementation into a simple system for atmospheric modelling, [SAM](http://rossby.msrc.sunysb.edu/~marat/SAM.html).
 
 The aims of this repository are to:
-1. develop a standalone fortran module based on this neural net that can be used elsewhere,
+1. provide a standalone fortran module of the convection parameterisation based on this neural net that can be used elsewhere,
 2. deploy the module in another atmospheric model, and
-3. evaluate its performance.
+3. evaluate its performance (and provide instructions/code to do this).
 
-We may also perform an investigation into interfacing the pytorch implementation of the Neural Net using the [pytorch-fortran bridging code](https://github.com/Cambridge-ICCS/fortran-pytorch-lib) developed at the [Institute of Computing for Climate Science](https://cambridge-iccs.github.io/).
+We have also started to perform an investigation into interfacing the pytorch implementation of the Neural Net using the [pytorch-fortran bridging code](https://github.com/Cambridge-ICCS/fortran-pytorch-lib) developed at the [Institute of Computing for Climate Science](https://cambridge-iccs.github.io/).
```

Comment:
I wonder if we just remove this for now?
No longer a focus for M2LInES and we could add it back later if we do something?

Reply:
But there is still some leftover code/NN weights?

Reply:
Not in the main repository that I can see, it's all in unmerged and stale (and messy) pull requests I think.